### PR TITLE
Display email on public profile

### LINKED ISSUE #3901: Show email of users in Pootle
Right now in many cases, it's impossible to know who is who on Pootle. In many occasions, I need to contact localizers in regards to an urgent project, but am unable to because people use nick names that do not correspond to any existing Mozillians profile and that do not appear on the l10n team page. Not having emails accessible prevents us from contacting these localizers in a timely manner. Requesting a fix for this :)

### PR DESCRIPTION
**Pull request by unho**
Picked from ac2b602b.

Fixes #3901.

### CODE REVIEW

**Comment by julen** [comment]
Even if this is admin-only, @iafan would appreciate your input on this.

File: pootle/templates/user/profile.html [hunk 1]
```diff
@@ -18,6 +18,15 @@ <h1 class="user-name">
         {{ object.display_name }}
       {% endif %}
     </h1>
+    {% if user.is_superuser or request.permissions.administrate %}
+    <p>
+      {% if object.email %}
+      <a href="mailto:{{ object.email }}">{{ object.email }}</a>
+      {% else %}
+      {% trans 'No email address.' %}
```
Comment (julen) [inline pootle/templates/user/profile.html]: Since we require email addresses, this is redundant.

Comment (unho) [inline pootle/templates/user/profile.html]: Thanks. Removing it.

**Comment by unho** [comment]
@iafan Can you please quickly review? Thanks in advance.

**Comment by julen** [comment]
Just to make things easier, @iafan here's how this looks like when admins visit any user profile page.
john_doe_profile

**Comment by iafan** [comment]
@unho, @julen, what happens when one clicks on that email?

**Comment by unho** [comment]
@iafan It is a regular `mailto` URL, so it should trigger the default handling for those URLs, which usually is opening the mail program (Thunderbird and such).

**Comment by iafan** [comment]
Ok, GTM.

**Pull request merged**